keps/sig-windows/2258-node-log-query/README.md

```diff
@@ -287,15 +286,42 @@ a 404 will be returned.
 
 ### Rollout, Upgrade and Rollback Planning
 
-_This section must be completed when targeting beta graduation to a release._
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+A rollout or rollback will not fail either when enabling or disabling the feature gate.
+The feature has no impact on running workloads.
```

Comment:
What we're asking about here is a rollout of the feature enablement. So, if your code panics, it will cause the kubelet to crash. Or, maybe it could cause kubelet to fail to come up, for example?

If they enable the feature gate and then kubelet starts failing, that would be a reason to rollback.